docs/beta/grain.md

```diff
@@ -1,85 +1,50 @@
----
-title: 🌾 Grain
-description: A reward for contribution
----
 
-SourceCred is both a social algorithm for assigning [cred](cred.md)
-scores, and a cryptoeconomic system for rewarding a project's contributors
-and sponsors. That cryptoeconomic system revolves around _grain_, a cred-based
-cryptoasset.
+# 🌾 How Grain Works
 
-The key properties of grain include:
+# 🌾 What is Grain? <br/>
+Grain is a token of value created by SourceCred. It can be used:<br/>
+1) to pay contributors<br/>
+2) to Boost contributions
 
-- **Grain is project-specific.**
+Let's go over each of these in depth.
 
-It's not a universal token; every project that is using SourceCred
-creates its own independent token. They can call it something other
-than grain (e.g. "Seeds") if they want.
 
-- **Grain is harvested by contributors, based on cred scores.**
+# 💰 Paying Contributors 
+### How do I get Grain?
 
-Grain is created during "harvests." In a harvest, a project mints new grain and
-distributes it to contributors based on their cred scores. How the grain is
-harvested and distributed is up to the project. For example, a project could
-mint a fixed amount of grain every week and distribute it to contributors based
-on their recent contributions, or it could have a target "grain-per-cred"
-payout ratio, and harvest to ensure that all contributors receive at least that
-amount.
+**CHANGE MONEY BAG EMOJI TO GRAIN TOKEN EMOJI IN LAST SLIDE: https://docs.google.com/presentation/d/1CIHI4xcDCiTgD2ACMG8Jnbv843Gq45V3SHKex915WY0/edit#slide=id.g8703f78739_0_409**
```

Comment:
*[//]: # (CHANGE MONEY BAG EMOJI TO GRAIN TOKEN EMOJI IN LAST SLIDE: https://docs.google.com/presentation/d/1CIHI4xcDCiTgD2ACMG8Jnbv843Gq45V3SHKex915WY0/edit#slide=id.g8703f78739_0_409)*

Use Dandelion's new comment syntax